src/pybind/mgr/cephadm/module.py

```diff
@@ -747,6 +747,17 @@ def _process_ls_output(self, host: str, ls: List[Dict[str, Any]]) -> None:
         self.cache.save_host(host)
         return None
 
+    def check_host_offline(self, host: str, kick_on_failure: bool = False) -> None:
```

Comment:
already done in 

https://github.com/ceph/ceph/blob/b55781d412f05e5ad99751cc4247a22d9ada5547/src/pybind/mgr/cephadm/serve.py#L292-L295

right?

Reply:
yup, just added the reset_con call to that and am now using it